### ISSUE TITLE
You are reading the rules wrong

### ISSUE THREAD

**Issue by mike-bailey** [CPFreeToolSpreader/Free-Cyberpatriot-Scripts#1]
First of all, these are my scripts. Second of all, use of free, open source scripts is not legal. Free, open source software is. Scripts you have to write.

**Comment by mike-bailey** [comment]
" Scripts. Only scripts created by the team’s COMPETITORS shall be stored on
electronic media for the online competition rounds. Scripts created by anyone other
than the Competitors shall NOT be stored on electronic media used during online
competition rounds or used by Competitors during the rounds. "

**Comment by CPFreeToolSpreader** [comment]
Open source Scripts are software. By your logic, we can not use most linux tools or anything that is written in python. It is clear that the rule is to stop a team having a mentor write them a script, but that goes away when it becomes open source. 

If the rule was to be taken 100% literally, then you can't use linux during competition unless you created linux. Did you make ubuntu?

**Comment by mike-bailey** [comment]
Open source scripts are not software. Fundamental computer science says there's a difference between scripts and tools. As the person who literally wrote everything in this repo, I'm telling you you're wrong. I've been talking to CPOC and they're considering revising the rule but that rule pretty clearly says only the team's competitors can write the scripts.

**Comment by CPFreeToolSpreader** [comment]
http://dictionary.reference.com/browse/software

scripts are not tools? "Tools" is not a technical term. again, how many of ubuntu's scripts did your team write?

guess you cant use this either: https://github.com/linuxsquad/Linux_Workstation_Harden_Security

Sans seems to think scripts are tools: http://www.sans.org/reading-room/whitepapers/linux/linux-scripts-monitor-security-197

how about this script: http://www.chkrootkit.org/

I don't want to call your team a rule breaking cheater, but if you don't remove the linux startup scripts before you start competition...

**Comment by mike-bailey** [comment]
I'm telling you right now you're reading the rules wrong. You don't have to believe me, I'm in college now. I taught like 14 teams and our team went to nationals so I'm fairly certain our team isn't a "rule breaking cheater". Please learn how CyberPatriot works before you compete.

**Comment by mike-bailey** [comment]
Correction, not 30. Forgot there was 16 in house ones. So closer to 30.

**Comment by CPFreeToolSpreader** [comment]
The rule is unenforceable and you are purposely avoiding the evidence I showed above that showed that scripts are tools, are software, and many acceptable tools we are allowed to use are scripts. Also it would be impossible for it to be a blanket ban on scripts if they allow those tools and even allow the use of linux. Stop trying to use an argument from authority by saying you have taught a bunch of teams and that you are in college.

**Comment by mike-bailey** [comment]
I'm saying "Look, I've been doing this, I've been teaching the rules, I've spoken specifically to the person who wrote the rule about the rule. I know the rules". That's an extremely valid argument. I know what software is and I know what a script is. You're just trying to justify it. If nothing else, you should probably respect the fact that this is my code and you took your own liberties in terms of open sourcing it (which isn't against any rules, it's just in bad faith) when it's now closed source since they're revising the rules.
You don't have to believe me, but if you don't think they can enforce their rules you have a very small scope of understanding about CyberPatriot.

**Comment by CPFreeToolSpreader** [comment]
if you didn't want it to not be open source, you should have used a private repo on bitbucket, and not have used github. This is taken from when your scripts were open source, maybe the one specificly on your computer is now closed source, this one isn't. This is open source. Actually, I seem to have some evidence now that this is the original.

**Comment by mike-bailey** [comment]
That what is the original... Lol

**Comment by mike-bailey** [comment]
Also, CCDC requires anything in use be open source but I asked people in the readme not fork it or copy it so I could maintain control. You didn't even fork it, you just ripped it.

**Comment by CPFreeToolSpreader** [comment]
once you make it open source, then its fair game for cyberpatriot. Maybe you shouldn't be using Cyberpatriot scripts for CCDC if they have open source rules. Also asking people in the readme to not fork or copy? Do you even open source bro. Please, can we solve this in court.

**Comment by CPFreeToolSpreader** [comment]
Also this is the original script from what I can tell, cant see any other ones from before this was uploaded.

**Comment by mike-bailey** [comment]
Yes I open source. I have like 14 open source projects... You didn't even do this under a legit account. You also literally went out of your way to not fork so maybe you're the one who's bad at open source. Not following the author's wishes doesn't make it a legal issue, it just makes you an asshole. Assholes in CS = Have fun going anywhere.

**Comment by mike-bailey** [comment]
I still have the repo from when I wrote it on Github, it's just private. It predates this repo. It also literally has my name in it. So. Don't try to take away my authorship, that's just a dick move.

**Comment by mike-bailey** [comment]
Also, once again, "once you make it open source, then its fair game for cyberpatriot" is not necessarily the rule.

**Comment by CPFreeToolSpreader** [comment]
my big problem in going places is that when customs sees my name, CPFreeToolSpreader, they simply dont believe me. You would be suprised how many times ive been searched at the airport

**Comment by mike-bailey** [comment]
What? Speak english bruh.

**Comment by CPFreeToolSpreader** [comment]
http://gyazo.com/f3bbf04c940917ea1b6f06b83cb7c0ff

IF it's open source and google-able then its an accepted software tool. again, please remind me how much of the ubuntu startup scripts did your team make?

**Comment by mike-bailey** [comment]
I've already told you what the rule is. Hilarious you're still arguing the interpretation of the rule when I'm telling you I've already spoken to the person who wrote that. Appropriate groups are now intervening on this account (Github and CyberPatriot) so it's no longer my problem.

**Comment by mike-bailey** [comment]
Really....? Why leave this open?

**Comment by CPFreeToolSpreader** [comment]
I am sorry, but you can not close this untell the issue is resolved. Please say that I am right so that we can call this a fixed issue.

**Comment by mike-bailey** [comment]
You're actually a five year old.

**Comment by CPFreeToolSpreader** [comment]
Thank you

**Comment by CPFreeToolSpreader** [comment]
I own this domain

**Comment by mike-bailey** [comment]
I'm actually botting the issue close so feel free to keep this up

**Comment by CPFreeToolSpreader** [comment]
You are being a child about this.

**Comment by mike-bailey** [comment]
You do realize I'm scripting this closing right... Are you actually pressing the button lmao

**Comment by cp-watcha** [comment]
I am also having this issue.

Can you help me figure out how to read the rules properly?

**Comment by CPFreeToolSpreader** [comment]
OH interesting, that is against the terms of service.

**Comment by CPFreeToolSpreader** [comment]
cp-watcha, are you one of mike-bailey's friends? Mike, are you sending fake accounts to spam me? I will let GitHub know about that.

**Comment by mike-bailey** [comment]
I'm actually not lol 
Also scripting isn't against TOS. Please actually read the TOS. There's a bandwidth cap but that's it. Scripting is insanely common in dev environments.

**Comment by cp-watcha** [comment]
I am one of his friend yes.  But i am not fake, can you help me understand how this rule works?

**Comment by CPFreeToolSpreader** [comment]
wow, great comment mike

**Comment by CPFreeToolSpreader** [comment]
stop trolling, mike

**Comment by cp-watcha** [comment]
Srs, please help me.

**Comment by mike-bailey** [comment]
Randomizing and increasing the time delay so by the time you go to sleep it just closes on you.

**Comment by CPFreeToolSpreader** [comment]
uh...that sounds like bullshit.
why would you randomize the time?

**Comment by mike-bailey** [comment]
What do you mean it's bullshit... It's like four lines. Can you even write code?

**Comment by CPFreeToolSpreader** [comment]
I only program in brainfuck

**Comment by mike-bailey** [comment]
Okay bye

**Comment by cp-watcha** [comment]
Can you guys help me please.

**Comment by CPFreeToolSpreader** [comment]
yeah, cp-watcha, what do you need help with?

**Comment by cp-watcha** [comment]
How do I read teh CP rules?

**Comment by CPFreeToolSpreader** [comment]
well, dont ask mike

**Comment by cp-watcha** [comment]
Who do I ask?

**Comment by CPFreeToolSpreader** [comment]
not mike
